<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wgt_QVideoPlayer</class>
 <widget class="QWidget" name="wgt_QVideoPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>360</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="SpacerH1">
     <property name="text">
      <string notr="true">SpacerH1</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="SpacerV1">
       <property name="text">
        <string notr="true">SpacerV1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QMovieLabel" name="MovieFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="SliderLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="VideoPlayerPlayPauseBT">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>32</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string extracomment="Tooltip in dialog box">Click to start play, click again to pause the movie</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton { /* all types of tool button */
 	 background: black;
     border-top: 1px solid #202020;
     border-bottom: 0px;
     border-left: 0px;
     border-right: 0px;
     border-radius: 0px;
     background-color: black;
 }

 QToolButton[popupMode=&quot;1&quot;] { /* only for MenuButtonPopup */
     padding-right: 10px; /* make way for the popup button */
 }

 QToolButton:pressed {
     background-color: black;
 }</string>
         </property>
         <property name="text">
          <string notr="true">...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/img/player_play.png</normaloff>:/img/player_play.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCustomRuller" name="CustomRuller">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>32</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string notr="true">CustomSlider</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="Position">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>14</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>14</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background: black;
subcontrol-origin: padding;
subcontrol-position: top right;
padding: 0px;
spacing: 2px;
border-top: 1px solid #202020;
border-bottom: 0px;
border-left: 0px;
border-right: 0px;
font: 8pt;
color: white;
</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <property name="text">
            <string notr="true">00:00:00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="BufferState">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>6</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>6</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Sans Serif</family>
             <pointsize>1</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QProgressBar:horizontal {
border: 0px;
border-radius: 0px;
background: black;
padding-top: 1px;
padding-bottom: 2px;
padding-left: 1px;
padding-right: 1px;
}
QProgressBar::chunk:horizontal {
background: red;
}</string>
           </property>
           <property name="maximum">
            <number>5</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="textVisible">
            <bool>false</bool>
           </property>
           <property name="format">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Duration">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>12</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background: black;
subcontrol-origin: padding;
subcontrol-position: top right;
padding: 0px;
spacing: 2px;
border-top: 0px;
border-bottom: 0px;
border-left: 0px;
border-right: 0px;
font: 8pt;
color: white;
</string>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <property name="text">
            <string notr="true">00:00:00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="SpacerH2">
     <property name="text">
      <string notr="true">SpacerH2</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomRuller</class>
   <extends>QLabel</extends>
   <header>../sharedfiles/QCustomRuller.h</header>
  </customwidget>
  <customwidget>
   <class>QMovieLabel</class>
   <extends>QLabel</extends>
   <header>wgt_QVideoPlayer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
